client/me/social-login/index.jsx

```diff
@@ -94,18 +93,40 @@ class SocialLogin extends Component {
 						{ errorUpdatingSocialConnection.message }
 					</Notice>
 				) }
+
 				<CompactCard>
 					{ translate(
-						'You’ll be able to log in faster by linking your WordPress.com account with your ' +
-							'social networks. We’ll never post without your permission.'
+						'You’ll be able to log in faster by linking your WordPress.com account with the following ' +
```

Comment:
Hi! I've found a possible matching string that has already been translated **24** times:
`translate( 'You’ll be able to log in faster by linking your WordPress.com account with your social networks. We’ll never post without your permission.' )` *ES Score: 6*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

Reply:
Indeed, let's not update the copy here 👍